docs/tutorials/plots.rst

```diff
@@ -51,15 +49,23 @@ These take, at minimum, `astroplan.Observer`, `astroplan.FixedTarget` and
 
 .. _plots_airmass:
 
-Airmass versus time plots are made in the following manner::
+Airmass vs time plots are made the following way:
+
+.. code-block:: python
 
-    plot_airmass(target, observer, time)
+    >>> from astroplan.plots import plot_airmass
+
+    >>> plot_airmass(target, observer, time)
```

Comment:
OK to skip.

Just to offer an alternative – you could also make this a full example that works, i.e. set up target, observer and time.

When I learn a new package I usually do it by reading through the docs, executing and playing with the examples given in an IPython notebook on the side. So I prefer to have as many working examples as possible, but I also realise that it's too much setup code in some cases which can be distracting for more narrative docs.